src/metabase/api/user.clj

```diff
@@ -66,26 +66,91 @@
 ;;; |                   Fetching Users -- GET /api/user, GET /api/user/current, GET /api/user/:id                    |
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
+(defn- status-clause
+  "Figure out what `where` clause to add to the user query when
+  we get a fiddly status and include_deactivated query.
+
+  This is to keep backwards compatibility with `include_deactivated` while adding `status."
+  [status include_deactivated]
+  (if include_deactivated
+    nil
+    (case status
+      "all" nil
+      "deactivated" [:= :is_active false]
+      "active" [:= :is_active true]
+      [:= :is_active true]
+      ))
+)
```

Comment:
nit: move this paren to the end of the previous line

Reply:
damn, I wish the linter would complain about this